examples/echo.rs

```diff
@@ -16,23 +17,20 @@ async fn echo(req: Request<Body>) -> Result<Response<Body>, hyper::Error> {
         // Simply echo the body back to the client.
         (&Method::POST, "/echo") => Ok(Response::new(req.into_body())),
 
-        // Convert to uppercase before sending back to client using a stream.
+        // Convert to uppercase before sending back to client.
         (&Method::POST, "/echo/uppercase") => {
-            let chunk_stream = req.into_body().map_ok(|chunk| {
-                chunk
-                    .iter()
-                    .map(|byte| byte.to_ascii_uppercase())
-                    .collect::<Vec<u8>>()
-            });
-            Ok(Response::new(Body::wrap_stream(chunk_stream)))
+            let body = hyper::body::aggregate(req.into_body()).await?;
```

Comment:
I think we do want to try to keep the body streaming... it may be possible to use [map_data](https://github.com/hyperium/http-body/blob/aa90d1de0b6fb7695037762ffbc3e7bb873eff22/http-body-util/src/lib.rs#L28), and then make it a `BoxBody`.

(A git dependency is fine for now.)

Reply:
Oh, actually, if https://github.com/hyperium/http-body/pull/63 merges, it might be easy to just replace most instances of a `StreamBody`.

Reply:
I wrote a TODO to fix this, as I was unable to fix it.